openverse_catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -25,200 +23,112 @@
 
 class PhylopicDataIngester(ProviderDataIngester):
     delay = 5
-    host = "http://phylopic.org"
-    # Use "base_endpoint" since class's "endpoint" parameter gets defined as a property
-    base_endpoint = f"{host}/api/a/image"
+    host = "https://www.phylopic.org"
+    endpoint = "https://api.phylopic.org/images"
     providers = {constants.IMAGE: prov.PHYLOPIC_DEFAULT_PROVIDER}
-    batch_limit = 25
 
-    @property
-    def endpoint(self) -> str:
-        """
-        Due to the way this DAG is run (via a dated range), **only one request is ever
-        issued** to retrieve all updated IDs. As such, it will only
-        return one endpoint.
-        """
-        list_endpoint = f"{self.base_endpoint}/list"
-        # Process for a given date
-        end_date = (date.fromisoformat(self.date) + timedelta(days=1)).isoformat()
-        # Get a list of objects uploaded/updated within a date range
-        # http://phylopic.org/api/#method-image-time-range
-        endpoint = f"{list_endpoint}/modified/{self.date}/{end_date}"
-        logger.info(f"Constructed endpoint: {endpoint}")
-        return endpoint
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.current_page = 1
+        self.total_pages = 0
+        self.build_param = 0
+
+    def ingest_records(self):
+        self._get_initial_query_params()
+        super().ingest_records()
+
+    def _get_initial_query_params(self) -> None:
+        """Get the required `build` param from the API and set the total pages."""
+        resp = self.get_response_json(query_params={})
+        if not resp:
+            raise Exception("No response from Phylopic API.")
```

Comment:
Maybe this might be a more specific exception?
```suggestion
            raise ValueError("No response from Phylopic API.")
```